app/views/shared/_run_table.slim

```diff
@@ -1,6 +1,7 @@
 - description ||= nil
 - cols ||= [:runner, :time, :name, :rival, :uploaded]
 - runs = order_runs(runs).page(params[:page]).includes(:user, :game)
+- all_runs = current_user.runs.order(:realtime_duration_ms) if cols.include?(:archived)
```

Comment:
if the run table is paginated, we are still grabbing all pages' worth of past runs with this line when we only need the current page's.